content/resources/help-centre/software-tips/github.md

```diff
@@ -0,0 +1,37 @@
+---
+title: GitHub
+---
+
+## What is it?
+
+[GitHub](https://github.com/) is a website and service which lets people store
+their code in the cloud, with a generous free option that has all the features
+you will need to get started. In simple terms, you can think of it as being like
+a DropBox for developers. But more than being just a place to store code, the
+value of GitHub lies in its version control capabilities, and open source
+collaborative community. 
+
```

Comment:
github also has tight integrations with zenodo which will allow researchers to mint DOIs for the software outputs